vault-mixin/dashboards/vault.json

```diff
@@ -2204,340 +2361,419 @@
+          "selected": true,
+          "text": [
+            "All"
+          ],
+          "value": [
+            "$__all"
+          ]
+        },
+        "datasource": {
+          "type": "prometheus",
+          "uid": "${datasource}"
+        },
+        "definition": "label_values(vault_core_active, job)",
+        "hide": 0,
+        "includeAll": true,
+        "label": "job",
+        "multi": true,
+        "name": "job",
         "options": [],
-        "query": "label_values(up{job=\"vault\"}, instance)",
+        "query": {
+          "query": "label_values(vault_core_active, job)",
+          "refId": "StandardVariableQuery"
+        },
         "refresh": 1,
-        "regex": "/([^:]+):.*/",
+        "regex": "",
         "skipUrlSync": false,
         "sort": 1,
         "tagValuesQuery": "",
-        "tags": [],
         "tagsQuery": "",
         "type": "query",
         "useTags": false
       },
       {
-        "allValue": null,
-        "current": {},
-        "datasource": "${datasource}",
-        "definition": "label_values(up{job=\"vault\",instance=~\"$node:(.*)\"}, instance)",
-        "hide": 2,
-        "includeAll": false,
-        "label": null,
-        "multi": false,
-        "name": "port",
+        "allValue": ".+",
+        "current": {
+          "selected": true,
+          "text": [
+            "All"
+          ],
+          "value": [
+            "$__all"
+          ]
+        },
+        "datasource": {
+          "type": "prometheus",
+          "uid": "${datasource}"
+        },
+        "definition": "label_values(vault_core_active{job=~\"$job\"}, instance)",
+        "hide": 0,
+        "includeAll": true,
+        "label": "instance",
+        "multi": true,
+        "name": "instance",
         "options": [],
-        "query": "label_values(up{job=\"vault\",instance=~\"$node:(.*)\"}, instance)",
+        "query": {
+          "query": "label_values(vault_core_active{job=~\"$job\"}, instance)",
+          "refId": "StandardVariableQuery"
+        },
         "refresh": 1,
-        "regex": "/[^:]+:(.*)/",
+        "regex": "",
         "skipUrlSync": false,
-        "sort": 0,
+        "sort": 1,
         "tagValuesQuery": "",
-        "tags": [],
         "tagsQuery": "",
         "type": "query",
         "useTags": false
       },
       {
         "allValue": "",
-        "current": {},
-        "datasource": "${datasource}",
-        "definition": "label_values(vault_secret_kv_count, mount_point)",
+        "current": {
+          "selected": true,
+          "text": [
+            "All"
+          ],
+          "value": [
+            "$__all"
+          ]
+        },
+        "datasource": {
+          "type": "prometheus",
+          "uid": "${datasource}"
+        },
+        "definition": "label_values(vault_secret_kv_count{job=~\"$job\",instance=~\"$instance\"}, mount_point)",
         "hide": 0,
         "includeAll": true,
-        "label": "Mount Point:",
+        "label": "",
```

Comment:
If we have labels for `job` and `instance` it makes sense to also have it for the custom variables

Reply:
updated.